Aztec/Classes/GUI/FormatBar/FormatBar.swift

```diff
@@ -634,22 +660,17 @@ private extension FormatBar {
         scrollView.alwaysBounceHorizontal = false
         scrollView.translatesAutoresizingMaskIntoConstraints = false
         scrollView.delegate = self
-
+        if layoutDirection == .rightToLeft {
+            scrollView.transform = CGAffineTransform(scaleX: -1, y: 1)
```

Comment:
Perhaps add a comment explaining what this does / why it's necessary?

Reply:
Also we have very similar code in two places... perhaps a private `UIView` helper method? `scrollView.transformForLayoutDirection` or `flipForLayoutDirection` or something? (with a well-named method, maybe the comment wouldn't be necessary :))